Source/Core/DolphinQt/Config/Graphics/GeneralWidget.cpp

```diff
@@ -330,8 +326,15 @@ void GeneralWidget::OnBackendChanged(const QString& backend_name)
   m_adapter_combo->setCurrentIndex(g_Config.iAdapter);
   m_adapter_combo->setEnabled(supports_adapters && !Core::IsRunning());
 
-  m_adapter_combo->setToolTip(supports_adapters ?
-                                  QString{} :
-                                  tr("%1 doesn't support this feature.")
-                                      .arg(tr(g_video_backend->GetDisplayName().c_str())));
+  constexpr char TR_ADAPTER_AVAILABLE_DESCRIPTION[] =
```

Comment:
```suggestion
  static constexpr char TR_ADAPTER_AVAILABLE_DESCRIPTION[] =
```

Otherwise, some compilers still attempt to construct the array on the stack at runtime.